tests/Couchbase.EntityFrameworkCore.FunctionalTests/ContosoUniversityTests/StudentTests.cs

```diff
@@ -61,94 +62,94 @@ public async Task Test_Get(string sortOrder,
+        var students = DbInitializer.someStudents();
+        await cleanup(students);
+        
+        context.Students.AddRange(students);
         var count = await context.SaveChangesAsync();
         Assert.Equal(8, count);
 
-        context.Students.RemoveRange(_students);
+        context.Students.RemoveRange(students);
         var count1 = await context.SaveChangesAsync();
-
         Assert.Equal(8, count1);
+        
         Assert.Equal(0, await context.Students.CountAsync());
     }
-    
+
     [Fact]
     public async Task Test_PersonStudent_UpdateRange()
     {
-        var context = _contosoFixture.DbContext;
-        context.Students.AddRange(_students);
+        var context = _contosoFixture.DbContext();
+        var students = DbInitializer.someStudents();
+        await cleanup(students);
 
-        var count = await context.SaveChangesAsync();
-      //  Assert.Equal(8, count);
 
-        var results = await context.Students.ToListAsync();
+        context.Students.AddRange(students);
+        var count = await context.SaveChangesAsync();
+        Assert.Equal(8, count);
 
-        context.Students.RemoveRange(_students);
+        context.Students.UpdateRange(students);
         var count1 = await context.SaveChangesAsync();
-
         Assert.Equal(8, count1);
+
+        context.Students.RemoveRange(students);
+        var count2 = await context.SaveChangesAsync();
+        Assert.Equal(8, count2);
+        
+        Assert.Equal(0, await context.Students.CountAsync());
     }
 
     #region Data
-    
-    Student[] _students =
-    [
-        new Student { FirstMidName = "Carson",   LastName = "Alexander",
-            EnrollmentDate = DateTime.Parse("2010-09-01"), ID = 8 },
-        new Student { FirstMidName = "Meredith", LastName = "Alonso",
-            EnrollmentDate = DateTime.Parse("2012-09-01"), ID = 1 },
-        new Student { FirstMidName = "Arturo",   LastName = "Anand",
-            EnrollmentDate = DateTime.Parse("2013-09-01"), ID = 2 },
-        new Student { FirstMidName = "Gytis",    LastName = "Barzdukas",
-            EnrollmentDate = DateTime.Parse("2012-09-01"), ID = 3 },
-        new Student { FirstMidName = "Yan",      LastName = "Li",
-            EnrollmentDate = DateTime.Parse("2012-09-01"), ID = 4 },
-        new Student { FirstMidName = "Peggy",    LastName = "Justice",
-            EnrollmentDate = DateTime.Parse("2011-09-01"), ID = 5 },
-        new Student { FirstMidName = "Laura",    LastName = "Norman",
-            EnrollmentDate = DateTime.Parse("2013-09-01"), ID = 6 },
-        new Student { FirstMidName = "Nino",     LastName = "Olivetto",
-            EnrollmentDate = DateTime.Parse("2005-09-01"), ID = 7 }
-    ];
-
     #endregion
 
     public async ValueTask DisposeAsync()
+    {
+    }
+
+    public void Dispose()
+    {
+        DisposeAsync().GetAwaiter().GetResult();
+    }
+
+    public async ValueTask cleanup(Student[] students)
```

Comment:
In c#, methods pascal cased and since its an async methond, the postfix should be "Async" like this: `public async ValueTask CleanupAsync(Student[] students)`